src/samarium/core.py

```diff
@@ -67,17 +67,17 @@ def import_module(data: str, reg: Registry) -> None:
         sys.stderr.write(dahlia("&4[RecursionError]\n"))
         return
     try:
-        path = sys.argv[1][: sys.argv[1].rfind("/") + 1] or "."
+        path = Path(sys.argv[1][: sys.argv[1].rfind("/") + 1] or ".")
     except IndexError:  # REPL
-        path = os.getcwd() + "/"
+        path = Path().absolute()
 
-    if f"{name}.sm" not in os.listdir(path):
+    if f"{name}.sm" not in list(path.iterdir()):
```

Comment:
Looks like this check is incorrect (`str` not in `list[PosixPath]`?). It makes local imports not work.